### PR TITLE
Script fix

### PR DESCRIPTION
**Pull request by dp247**
lpod
Added in a user agent header to the session to stop the 403 errors from letting BeautifulSoup do its beautiful work. Also added in a pause since I know from experience (not this script, JPod's own mass downloader for podcast applications) that you can get IP-banned for requesting too many file downloads. May have corrected a comment here or there too...

### CODE REVIEW

File: language101_scraper.py [hunk 1]
```diff
@@ -12,20 +13,20 @@
...
-
 USERNAME = args.username or input('Username(mail): ')
 PASSWORD = args.password or input('Password: ')
 COURSE_URL = args.url or input('Please insert first lesson URL of the desired course, for example:\n'
-    ' * https://www.japanesepod101.com/lesson/lower-beginner-1-a-formal-japanese-introduction/?lp=116\n'
-    ' * https://www.spanishpod101.com/lesson/basic-bootcamp-1-a-pleasure-to-meet-you/?lp=425\n'
-    ' * https://www.chineseclass101.com/lesson/absolute-beginner-1-meeting-whats-your-name/?lp=208\n')
+                               ' * https://www.japanesepod101.com/lesson/lower-beginner-1-a-formal-japanese-introduction/?lp=116\n'
+                               ' * https://www.spanishpod101.com/lesson/basic-bootcamp-1-a-pleasure-to-meet-you/?lp=425\n'
+                               ' * https://www.chineseclass101.com/lesson/absolute-beginner-1-meeting-whats-your-name/?lp=208\n')
```
Comment (michalcichon) [inline language101_scraper.py:29]:
```suggestion
    ' * https://www.japanesepod101.com/lesson/lower-beginner-1-a-formal-japanese-introduction/?lp=116\n'
    ' * https://www.spanishpod101.com/lesson/basic-bootcamp-1-a-pleasure-to-meet-you/?lp=425\n'
    ' * https://www.chineseclass101.com/lesson/absolute-beginner-1-meeting-whats-your-name/?lp=208\n')
```
Why do you need these extra spaces?

File: README.md [hunk 2]
```diff
@@ -131,6 +131,6 @@ The license below refers only to the script and not to the downloaded content.
 
 ![broken](readme/status.gif)
 
-It's broken, I will fix it, maybe...
+~~It's broken, I will fix it, maybe...~~
```
Comment (michalcichon) [inline README.md:134]: I would remove it totally from the README. No need to mention at all. What we could place here is that the script was fixed + date when we fixed it.

**Review by michalcichon — APPROVED** [review]
Looks good 🙂

**Comment by nedlir** [comment]
Made few tests, it does work.

Good job David.

**Comment by dp247** [comment]
No problem, happy to help! :)

**Pull request merged**